### PR TITLE
Initial modular UI complete

### PR DESCRIPTION
**Pull request by MrBanananator**

### CODE REVIEW

File: GoogleMeetup/src/main/webapp/style.css [hunk 1]
```diff
@@ -1,9 +1,353 @@
...
+
+#logo .red {
+    width: 81%;
+    height: 50%;
+    position: absolute;
+    top: 0px;
+    background: #f44336;
+}
+
+
```
Comment (jg-g) [inline GoogleMeetup/src/main/webapp/style.css:340]: Do we need so many blank lines?

File: GoogleMeetup/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,9 +1,353 @@
...
+#navbar.active li:nth-of-type(2) a { animation: topItemAnimation 0.5s ease 0.16666666666666666s backwards;}
+#navbar.active li:nth-of-type(3) a { animation: topItemAnimation 0.5s ease 0.25s backwards;}
+#navbar.active li:nth-of-type(4) a { animation: topItemAnimation 0.5s ease 0.3333333333333333s backwards;}
+#navbar.active li:nth-of-type(5) a { animation: topItemAnimation 0.5s ease 0.4166666666666667s backwards;}@media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) { #navbar.active li a{animation:none!important;}}
+
+
+
+
+
+
```
Comment (jg-g) [inline GoogleMeetup/src/main/webapp/style.css:207]: Given the way you've structured the file, it feels like these should be multiple CSS files.

File: GoogleMeetup/src/main/webapp/style.css (hunk 1 above)
Comment (MrBanananator) [reply GoogleMeetup/src/main/webapp/style.css:340]: Yeah not really, I was just keeping things separate while I was working on the different parts. I'll remove them.

File: GoogleMeetup/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,18 +1,97 @@
...
+
+                <!-- <li class="full-width">
+                    <span>Something <i class="arrow"></i></span>
+                    <div class="dropdown">
+                        <div class="clm">
+                            <h3>Integer</h3>
+                            <a href="#">Lacus iaculis</a>
+                            <a href="#">Eu tortor</a>
+                            <a href="#">Luctus varius</a>
+                        </div>
```
Comment (jg-g) [inline GoogleMeetup/src/main/webapp/index.html:75]: These can be deleted as they're not being used.

**Pull request closed without merging**